chore(workspace): playwright dev mode (#225)
Vonage/vivid-3: docs/ui-tests/readme.md

Each component folder should contain a `ui.test.ts` file.

> The file must be updated with the `componentName` and snapshot path `'./snapshots/componentName.png'`.

Reviewer: this are individual variables. can this in the future be part of a configurable object? maybe also have style property to override etc'.

Author: What? Can't relate it to this one...

> PRO TIP: When you generate a new component using `npm run nx workspace-generator vivid-component my-component` you will get this file populated for you to get you up and running with ui-tests.

Reviewer: ```suggestion
PRO TIP: utilizing the _component_ generator (`npm run nx workspace-generator vivid-component my-component`) will populate basic `ui.test.ts` to get you up and running with ui-tests.
```
